src/main/java/seedu/address/commons/util/DateUtil.java

```diff
@@ -35,4 +38,15 @@ public static List<LocalDate> convertToDateList(String args) throws ParseExcepti
         }
         return dateList;
     }
+
+    /**
+     * Returns a formatted local date in the form of month date, year, time.
+     *
+     * @param appointmentDateTime the date and time to be formatted.
+     * @return a formatted string.
+     */
+    public static String formatDateTimeForDisplay(LocalDateTime appointmentDateTime) {
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("MMMM dd, yyyy, hh:mm a").withLocale(Locale.US);
```

Comment:
Would it be better to have a public variable `formatter`, which is used universally by all things needing a formatter?